tests/test_doctest_ufunc.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+import sys
+import glob
+
+pytest_plugins = ['pytester']
+
```

Comment:
If we'd go with my suggestion to not rely on `numpy`, I guess this would need a `pytest.importorskip('numpy')` stanza.

Reply:
Sure. Are we going with Numpy as optional?

Reply:
OK, done.